cmake/config/OpenVDBVersions.cmake

```diff
@@ -34,24 +34,23 @@ if(NOT DISABLE_DEPENDENCY_VERSION_CHECKS)
   # What's usually important is that the version of libstdc++ and glibc in use
   # matches. Compilers other than GCC should provide options to ensure this
   # targetting e.g. --gcc-toolchain and -fgnuc-version with Clang.
-  set(MINIMUM_GCC_VERSION 9.3.1)
+  set(MINIMUM_GCC_VERSION 11.2.1)
   set(MINIMUM_CLANG_VERSION 5.0)
   set(MINIMUM_ICC_VERSION 19)
-  set(MINIMUM_MSVC_VERSION 19.28) # 1928 (Visual Studio 2019 Version 16.8 + 16.9)
+  set(MINIMUM_MSVC_VERSION 19.30) # 1928 (Visual Studio 2019 Version 16.8 + 16.9)
 
-  # Should be 1.76 for VFX 22, but only version in apt is 1.73
-  set(MINIMUM_BOOST_VERSION 1.73)
+  set(MINIMUM_BOOST_VERSION 1.80)
   set(MINIMUM_PYBIND_VERSION 2.9.1)
   set(MINIMUM_IMATH_VERSION 3.1)
   set(MINIMUM_OPENEXR_VERSION 3.1)
   set(MINIMUM_ZLIB_VERSION 1.2.7)
   set(MINIMUM_TBB_VERSION 2020.3)
-  set(MINIMUM_LLVM_VERSION 10.0.0)
+  set(MINIMUM_LLVM_VERSION 13.0.0)
   set(MINIMUM_BLOSC_VERSION 1.17.0)
   set(MINIMUM_GLFW_VERSION 3.1)
 
-  set(MINIMUM_PYTHON_VERSION 3.9.1)
-  set(MINIMUM_NUMPY_VERSION 1.20.0)
+  set(MINIMUM_PYTHON_VERSION 3.10)
+  set(MINIMUM_NUMPY_VERSION 1.23.0)
 
```

Comment:
Do we need to add `MINIMUM_NANOBIND_VERSION` here?

Reply:
I just tried that, but it seems that nanobind does not store the version number in `2.0.0`, so I'll revert that change.

This issue was reported here (https://github.com/wjakob/nanobind/pull/618) and fixed in `2.1.0`, so I'll document the future minimum version as that and we can bump it next year.

Reply:
Thanks for trying this. You're right, seems that adding a requirement for NanoBind version 2.0.0 breaks a few of the CIs.